pkg/operator/operator.go

```diff
@@ -138,17 +138,36 @@ func (optr *Operator) Run(workers int, stopCh <-chan struct{}) {
 	defer utilruntime.HandleCrash()
 	defer optr.queue.ShutDown()
 
-	glog.Info("Starting MachineAPIOperator")
+	glog.Info("Starting MachineAPIO--rator")
 	defer glog.Info("Shutting down MachineAPIOperator")
 
+	// TODO(alberto) operatorConfig as CRD?
+	operatorConfig, err := render.Config(optr.config)
+	if err != nil {
+		glog.Fatalf("Failed getting operatorConfig: %v", err)
+		return
+	}
+	glog.Infof("operator config %+v", operatorConfig)
+	err = optr.syncClusterAPIServer(*operatorConfig)
+	if err != nil {
+		glog.Fatalf("Failed sync-up cluster apiserver: %v", err)
+		return
+	}
+	glog.Info("Synched up cluster api server")
+	err = optr.syncClusterAPIController(*operatorConfig)
+	if err != nil {
+		glog.Fatalf("Failed sync-up cluster api controller: %v", err)
+		return
+	}
+	glog.Info("Synched up cluster api controller")
 	if !cache.WaitForCacheSync(stopCh,
 		optr.crdListerSynced,
```

Comment:
if we do https://github.com/openshift/machine-api-operator/pull/60/files#r218747267
then we just wait for deployments (aggregated server/controllers here)